test/rules/prefer-conditional/test.ts.lint

```diff
@@ -0,0 +1,37 @@
+let x;
+if (true) {
+~~ [X]
+    x = 1;
+} else {
+    x = 2;
+}
+
+if (true)
+~~ [X]
+    x = "TRUE";
```

Comment:
what happens if there is complex logic in these branches? I would prefer to use this more verbose pattern over a ternary if switching to a ternary meant a very long multiline statement.